### PR TITLE
promote preprod changes to prod

### PR DESCRIPTION
**Pull request by haraprasadj**
Link to Jira ticket if there is one:

### CODE REVIEW

**Comment by PlanXCyborg** [comment]
# healdata.org/manifest.json
## :warning: Services on branch
- metadata
- portal
## Deployment changes
- fence
  - By default Fence will allow only visas issued by RAS staging, RAS prod, and itself; if a different set of allowed visa issuers is needed, edit the GA4GH_VISA_ISSUER_ALLOWLIST config value. Otherwise, no action needed. (uc-cdis/fence#933)
  - /link/google will DENY requests with "redirect" param values that aren't in the "allowed redirects" for OAuth2/OIDC Clients in Fence (You have to add any valid redirects to the client before deploying this) (uc-cdis/fence#927)

**Comment by PlanXCyborg** [comment]
# healdata.org/manifest.json
## :warning: Services on branch
- metadata
- portal
## Deployment changes
- fence
  - By default Fence will allow only visas issued by RAS staging, RAS prod, and itself; if a different set of allowed visa issuers is needed, edit the GA4GH_VISA_ISSUER_ALLOWLIST config value. Otherwise, no action needed. (uc-cdis/fence#933)
  - /link/google will DENY requests with "redirect" param values that aren't in the "allowed redirects" for OAuth2/OIDC Clients in Fence (You have to add any valid redirects to the client before deploying this) (uc-cdis/fence#927)

File: healdata.org/manifest.json [hunk 1]
```diff
@@ -11,24 +11,24 @@
...
-    "portal": "quay.io/cdis/data-portal:feat_heal-mvp",
+    "portal": "quay.io/cdis/data-portal:feat_heal",
     "revproxy": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/nginx:2021.07",
     "sheepdog": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/sheepdog:2021.07",
     "ssjdispatcher": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/ssjdispatcher:2021.07",
     "sower": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/sower:2021.07",
     "spark": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/gen3-spark:2021.07",
     "tube": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/tube:2021.07",
-    "wts": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/workspace-token-service:2021.06"
+    "wts": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/workspace-token-service:2021.07"
```
Comment (johnfrancismccann) [inline healdata.org/manifest.json:31]: Can we keep WTS pinned to `2021.06`? Please see [recent changes](https://github.com/uc-cdis/cdis-manifest/pull/3229).

**Comment by PlanXCyborg** [comment]
# healdata.org/manifest.json
## :warning: Services on branch
- metadata
- portal
## Deployment changes
- fence
  - By default Fence will allow only visas issued by RAS staging, RAS prod, and itself; if a different set of allowed visa issuers is needed, edit the GA4GH_VISA_ISSUER_ALLOWLIST config value. Otherwise, no action needed. (uc-cdis/fence#933)
  - /link/google will DENY requests with "redirect" param values that aren't in the "allowed redirects" for OAuth2/OIDC Clients in Fence (You have to add any valid redirects to the client before deploying this) (uc-cdis/fence#927)

**Comment by PlanXCyborg** [comment]
# healdata.org/manifest.json
## :warning: Services on branch
- metadata
- portal
## Deployment changes
- fence
  - By default Fence will allow only visas issued by RAS staging, RAS prod, and itself; if a different set of allowed visa issuers is needed, edit the GA4GH_VISA_ISSUER_ALLOWLIST config value. Otherwise, no action needed. (uc-cdis/fence#933)
  - /link/google will DENY requests with "redirect" param values that aren't in the "allowed redirects" for OAuth2/OIDC Clients in Fence (You have to add any valid redirects to the client before deploying this) (uc-cdis/fence#927)

**Review by nmetokishlubsky — APPROVED** [review]

**Review by nmetokishlubsky — APPROVED** [review]

**Review by jmontmaxwell — APPROVED** [review]

**Pull request merged**